milestone2.ts

```diff
@@ -164,9 +190,16 @@ let getVolunteerInfo: GetVolunteerInfo = (vol) => {
 /* these criteria, return undefined
  */
 
-let daBigTest: GetVolunteer; // Code here
+let daBigTest: GetVolunteer = (data) => {
+  console.log(data)
```

Comment:
make sure to remove unnecessary print statements (which you may have used for testing) from the functions before submitting